src/main/java/com/wynntils/modules/map/overlays/ui/SeaskipperWorldMapUI.java

```diff
@@ -119,24 +138,28 @@ private void updateItems() {
 
             if (stack.isEmpty() || !stack.hasDisplayName()) continue;
 
-            if (stack.getItem() == Items.NAME_TAG) receivedItems = true;
-            else continue;
+            if (stack.getItem() == Items.NAME_TAG) {
+                receivedItems = true;
 
-            String displayname = getTextWithoutFormattingCodes(stack.getDisplayName());
+                String displayname = getTextWithoutFormattingCodes(stack.getDisplayName());
 
-            if (displayname == null) continue;
+                if (displayname == null) continue;
 
-            Matcher result = SEASKIPPER_PASS_MATCHER.matcher(displayname);
-            if (!result.find()) {
-                Reference.LOGGER.info("Seaskipper Pass with name" + stack.getDisplayName() + " didn't parse");
-                continue;
-            }
+                Matcher result = SEASKIPPER_PASS_MATCHER.matcher(displayname);
+                if (!result.find()) {
+                    Reference.LOGGER.info("Seaskipper Pass with name" + stack.getDisplayName() + " didn't parse");
+                    continue;
+                }
 
```

Comment:
Just to avoid crashes if something changes, I would recommend checking if the amount of groups is equivalent to the ones you need.

```suggestion
				if (result.groupCount() != 2) return;
				
```